EventsExpress/ClientApp/src/components/inventory/VisitorSeeItem.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import IconButton from "@material-ui/core/IconButton";
+import Tooltip from '@material-ui/core/Tooltip';
+
+export default class VisitorSeeItem extends Component {
+    render() {
+        const { item, disabledEdit, onWillNotTake, markItemAsEdit, markItemAsWillTake, usersInventories, user } = this.props;
+        return (
+            <>
+                {!item.isEdit &&
+                    <>
+                        <div className="col col-md-4 d-flex align-items-center">
+                            <span className="item" onClick={() => this.onAlreadyGet(item)}>{item.itemName}</span>
+                        </div>
+                        <div className="col d-flex align-items-center" key={item.id}>
+                                {item.showAlreadyGetDetailed &&
+                                    usersInventories.data.map(data => {
+                                        return (
+                                            data.inventoryId === item.id 
+                                            ?   <div>{data.user.name}: {data.quantity};</div>
+                                            :   null
+                                        );
+                                    })
+                                }
+
+                                {!item.showAlreadyGetDetailed && 
+                                    <>
+                                        {usersInventories.data.length === 0 ? 
+                                                0 
+                                                : usersInventories.data.reduce((acc, cur) => {
```

Comment:
Duplication of code in the parent component and down below as well - consider doing it in single place.